packages/truffle-migrate/index.js

```diff
@@ -148,6 +148,35 @@ const Migrate = {
     };
   },
 
+  wrapLegacyProvider: function(provider, logger) {
```

Comment:
Looks like `wrapLegacyProvider` is no longer used anywhere. I think we can delete it.